swap_meet/vendor.py

```diff
@@ -1,2 +1,139 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory = None):
+        if inventory is not None:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+#add new item and return inventory list
+    def add(self,add_item):
+        self.inventory.append(add_item)
+        return add_item
+
+#remove duplicates from inventory list if any, else return None
```

Comment:
I don't know that this comment still matches the function behavior.  I don't think it removes duplicates. Rather, it removes an item from the inventory if it existed in the inventory :)